lib/view/comic_page.dart

```diff
@@ -132,7 +133,7 @@ class ComicPage extends StatelessWidget {
         FlatButton(
           child: const Text(
             'Перейти',
-            style: TextStyle(color: Colors.teal),
+            style: TextStyle(color: app_styles.kPrimaryColor),
           ),
```

Comment:
But wouldn't it be better to use Theme.of and depend on actual theme color?

It would also mean that we can easily implement multiple themes in future.